strings/traumas.json

```diff
@@ -118,7 +132,9 @@
         "mr terry",
         "berry",
         "vent hell",
-        "colonel hall"
+        "colonel hall",
+		"manwell",
+		"manual"
```

Comment:
shouldnt it be mr manuel

Reply:
*shrug

Reply:
Should probably be added just to match up with everything else